tests/utils.rs

```diff
@@ -42,22 +73,21 @@ pub fn test_run_with_tempfile(
     Ok((exitcode, context.get_captured_stdout()))
 }
```

Comment:
Could `test_run_with_tempfile` now be changed to not return `(ExitCode, String)` and instead take an argument of type `Expect`?

Reply:
As we discussed passing an `Expect` to `test_run_with_tempfile` does not work out. But I simplified `test_run_with_tempfile` to return just an exit code.